test/e2e/tests/tokens/custom-token-send-transfer.spec.js

```diff
@@ -136,6 +136,9 @@ describe('Transfer custom tokens @no-mmi', function () {
           text: '-1.5 TST',
         });
 
+        // this delay helps prevent flakiness. it allows driver to wait until send transer is "confirmed"
+        await driver.delay(5000);
```

Comment:
Is there a selector we can poll for here?  We really want to get away from these types of generic delays.

Reply:
Thanks for highlighting this. I added a selector to wait for the `Confirmed` text rather than generically delaying for an arbitrary amount of time: https://github.com/MetaMask/metamask-extension/pull/27184/commits/b1266e7c3764e85b73ccd95c55990ebd504aa03a